README.md

```diff
@@ -1,4 +1,4 @@
-# Django Name [![Build Status](https://travis-ci.org/unt-libraries/django-name.svg?branch=master)](https://travis-ci.org/unt-libraries/django-name) [![Docs Status](https://img.shields.io/badge/docs-latest-blue.svg)](https://django-name.readthedocs.org) [![PyPI Version](https://img.shields.io/pypi/v/django-name.svg)](https://pypi.python.org/pypi/django-name)
+# Django Name [![Build Status](https://github.com/unt-libraries/django-name/actions/workflows/test.yml/badge.svg?branch=master)](https://travis-ci.org/unt-libraries/django-name/actions) [![Docs Status](https://img.shields.io/badge/docs-latest-blue.svg)](https://django-name.readthedocs.org) [![PyPI Version](https://img.shields.io/pypi/v/django-name.svg)](https://pypi.python.org/pypi/django-name)
```

Comment:
shouldn't the other travis-ci.org URL be changed here too?

Reply:
Yes, it should. Good catch!